package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "maintheme",
```

Comment:
I would add organization scope to the package name: `@nbb/maintheme`
https://docs.npmjs.com/misc/scope